README.md

```diff
@@ -34,9 +34,11 @@ over its REST API.
 This version of Hibernate Search requires:
 
 * Java SE 8
-* Hibernate ORM 5.2.x
+* Hibernate ORM 5.2.3.Final or a later 5.2.x
```

Comment:
I confirm, it should work fine since 5.2.3+. I actually tested the whole range ;)